wallet/how-to/use-non-evm-networks/starknet/tutorial.md

```diff
@@ -0,0 +1,325 @@
+---
+description: Starknet integration tutorial
+sidebar_position: 6
+---
+
+# Build a Basic dApp with get-starknet and React TypeScript
+
+In this tutorial, you'll learn how to set up a basic dApp that uses get-starknet to connect to MetaMask and display the user's wallet address.
+
+## 1. Project Setup
```

Comment:
```suggestion
## 1. Set up the project
```

It looks like this tutorial might be copy-paste and needs an editing pass? I'll refrain from adding more comments until it's more ready. Headings should be sentence case and avoid gerunds. Admonitions should be properly formatted, etc.